packages/ncurses/build.sh

```diff
@@ -61,65 +66,72 @@ share/man/man5
+		ln -sfr "libncursesw.so.${TERMUX_PKG_VERSION:0:3}" "lib${lib}.so.${TERMUX_PKG_VERSION:0:1}"
+		ln -sfr "libncursesw.so.${TERMUX_PKG_VERSION:0:3}" "lib${lib}.so"
+		ln -sfr libncursesw.a "lib${lib}.a"
+		(cd pkgconfig; ln -sfr ncursesw.pc "${lib}.pc") || termux_error_exit "Failed to install legacy comatibility symlink for '${lib}'"
 	done
 
 	# Some packages want these:
-	cd $TERMUX_PREFIX/include/
+	cd "$TERMUX_PREFIX/include/" || termux_error_exit "Prefix 'include' directory does not exist."
 	rm -Rf ncurses{,w}
 	mkdir ncurses{,w}
 	ln -s ../{curses.h,eti.h,form.h,menu.h,ncurses_dll.h,ncurses.h,panel.h,termcap.h,term_entry.h,term.h,unctrl.h} ncurses
 	ln -s ../{curses.h,eti.h,form.h,menu.h,ncurses_dll.h,ncurses.h,panel.h,termcap.h,term_entry.h,term.h,unctrl.h} ncursesw
 
 	# Strip away 30 years of cruft to decrease size.
-	local TI=$TERMUX_PREFIX/share/terminfo
-	mv $TI $TERMUX_PKG_TMPDIR/full-terminfo
-	mkdir -p $TI/{a,d,e,g,n,k,l,p,r,s,t,v,x}
-	cp $TERMUX_PKG_TMPDIR/full-terminfo/a/ansi $TI/a/
-	cp $TERMUX_PKG_TMPDIR/full-terminfo/d/{dtterm,dumb} $TI/d/
-	cp $TERMUX_PKG_TMPDIR/full-terminfo/e/eterm-color $TI/e/
-	cp $TERMUX_PKG_TMPDIR/full-terminfo/g/gnome{,-256color} $TI/g/
-	cp $TERMUX_PKG_TMPDIR/full-terminfo/n/nsterm $TI/n/
-	cp $TERMUX_PKG_TMPDIR/full-terminfo/k/kitty{,+common,-direct} $TI/k/
-	cp $TERMUX_PKG_TMPDIR/full-terminfo/l/linux $TI/l/
-	cp $TERMUX_PKG_TMPDIR/full-terminfo/p/putty{,-256color} $TI/p/
-	cp $TERMUX_PKG_TMPDIR/full-terminfo/r/rxvt{,-256color} $TI/r/
-	cp $TERMUX_PKG_TMPDIR/full-terminfo/s/{screen{,2,-256color},st{,-256color}} $TI/s/
-	cp $TERMUX_PKG_TMPDIR/full-terminfo/t/tmux{,-256color} $TI/t/
-	cp $TERMUX_PKG_TMPDIR/full-terminfo/v/{vt52,vt100,vt102} $TI/v/
-	cp $TERMUX_PKG_TMPDIR/full-terminfo/x/xterm{,-color,-new,-16color,-256color,+256color} $TI/x/
+	local TI="$TERMUX_PREFIX/share/terminfo"
+	mv "$TI" "$TERMUX_PKG_TMPDIR/full-terminfo"
+	mkdir -p "$TI"/{a,d,e,f,g,n,k,l,p,r,s,t,v,x}
+	cp "$TERMUX_PKG_TMPDIR"/full-terminfo/a/{alacritty{,+common,-direct},ansi} "$TI/a/"
+	cp "$TERMUX_PKG_TMPDIR"/full-terminfo/d/{dtterm,dumb} "$TI/d/"
+	cp "$TERMUX_PKG_TMPDIR"/full-terminfo/e/eterm-color "$TI/e/"
+	cp "$TERMUX_PKG_TMPDIR"/full-terminfo/f/foot{,+base,-direct} "$TI/f/"
+	cp "$TERMUX_PKG_TMPDIR"/full-terminfo/g/gnome{,-256color} "$TI/g/"
+	cp "$TERMUX_PKG_TMPDIR"/full-terminfo/n/nsterm "$TI/n/"
+	cp "$TERMUX_PKG_TMPDIR"/full-terminfo/k/kitty{,+common,-direct} "$TI/k/"
+	cp "$TERMUX_PKG_TMPDIR"/full-terminfo/l/linux "$TI/l/"
+	cp "$TERMUX_PKG_TMPDIR"/full-terminfo/p/putty{,-256color} "$TI/p/"
+	cp "$TERMUX_PKG_TMPDIR"/full-terminfo/r/rxvt{,-256color} "$TI/r/"
+	cp "$TERMUX_PKG_TMPDIR"/full-terminfo/s/{screen{,2,-256color},st{,-256color}} "$TI/s/"
+	cp "$TERMUX_PKG_TMPDIR"/full-terminfo/t/tmux{,-256color} "$TI/t/"
+	cp "$TERMUX_PKG_TMPDIR"/full-terminfo/v/vt{52,100,102} "$TI/v/"
+	cp "$TERMUX_PKG_TMPDIR"/full-terminfo/x/xterm{,-color,-new,-16color,-256color,+256color} "$TI/x/"
+
```

Comment:
Are you listing all possible terminals here? Because there are kermit and xfce4-terminal unlisted.

Reply:
And mate-terminal too. I am not sure these terminals need their own terminfo.

Reply:
I didn't add any here compared to what we ship currently.
But all three of those terminals are VTE based, so they shouldn't need a bespoke terminfo.
The `gnome` one should apply.